packages/playground-server/scripts/postinstall.sh

```diff
@@ -5,4 +5,5 @@ set -e
 if [ ! -z "${IS_HEROKU_ENV}" ]; then
     echo "📟  Detected Heroku. Transpiling Typescript into JS for Node.JS to run on server."
     tsc -p tsconfig.heroku.json &> /dev/null || :
```

Comment:
Also what's the point of directing the output of `tsc` to `/dev/null`? AFAICT doing anything after the `tsc` command itself seems to be irrelevant (and possibly erroneous).

Reply:
Several build warning show up even if it is successful which are annoying and we can debug it offline